src/Badge.js

```diff
@@ -5,12 +5,16 @@ class Badge extends React.Component {
         children: PropTypes.oneOfType([
             PropTypes.element,
             PropTypes.string
-        ]).isRequired,
+        ]),
         text: PropTypes.string.isRequired
     }
 
     render() {
         var { children } = this.props;
+
+        // No badge if no children
+        if(!React.Children.count(children)) return null;
```

Comment:
I'm talking about this line.
It seems like stateless components do not support returning null from render.
Current test suite skips it somehow.